src/display/editor/ink.js

```diff
@@ -478,10 +566,14 @@ class InkEditor extends AnnotationEditor {
     }
     this.#setStroke();
 
-    const { canvas, ctx } = this;
+    const {
+      canvas: { height, width },
+      ctx,
+    } = this;
     ctx.setTransform(1, 0, 0, 1, 0, 0);
-    ctx.clearRect(0, 0, canvas.width, canvas.height);
+    ctx.clearRect(0, 0, width, height);
     this.#updateTransform();
+
```

Comment:
These changes look unneeded, since I don't believe that any logical changes happened here?